app/views/admin/dashboard/index.html.erb

```diff
@@ -0,0 +1,3 @@
+<div id="dashboard-header">
```

Comment:
do you want to change the name of the div id to something more like

`<div id="admin-header">` or `<div id="admin-dashboard-header">`

The reason is that we also have a Merchants dashboard header that is different than the Admin dashboard header.